src/kivi/ast/src/ast/parsing_context.cc

```diff
@@ -23,9 +23,13 @@ namespace syntax_analyzer
 		m_all_scopes.emplace_back();
 	}
 
-	void parsing_context::exit_scope()
+	void parsing_context::exit_scope() noexcept(false)
```

Comment:
whyyy? why `noexcept(false)`: https://stackoverflow.com/questions/16244313/does-adding-noexceptfalse-benefit-the-code-in-any-way

Reply:
**_Update:_** to be decided